amundsen_application/static/package-lock.json

```diff
@@ -1074,13 +1074,15 @@
           "version": "1.0.0",
```

Comment:
Did you change your local npm/node versions recently?

Reply:
I upgraded to `node v10.15.3` and `npm 6.9.0` a while ago when there were some github issues related node compatibility. node 8 is on maintenance mode while node 10 is active.